src/components/mainBanner/MainBanner.tsx

```diff
@@ -12,11 +14,14 @@ const MainBanner: React.FC = () => {
             .then(res => setFilmInfo(res as FilmInfoInterface))

```

Comment:
do not use typescript's "as" , instead create an initial state, matching the interface, pass it as initial state to the useState instead of an empty object and remove "as FilmInfoInterface"